Source/Controllers/Preferences/Panes/SPNetworkPreferencePane.m

```diff
@@ -194,7 +254,18 @@ - (IBAction)updateSSHConfig:(NSPopUpButton *)sender
 	// choose a config file not listed
 	if ((NSUInteger) [sshConfigChooser indexOfSelectedItem] == ([[sshConfigChooser itemArray] count] - 1)) {
 		// open the file chooser dialog
-		[self chooseSSHConfig];
+        PanelOptions *options = [[PanelOptions alloc] init];
+
+        options.allowsMultipleSelection = YES;
```

Comment:
Can you really select multiple ssh config files? Why more than one?

Reply:
one can include another .. need bookmarks for all